.ci/jobDSL/jobs/apm-ci/apm-shared/oblt-test-env/stack_docker_images.groovy

```diff
@@ -30,7 +30,7 @@ pipelineJob("apm-shared/oblt-test-env/stack-docker-images") {
     artifactDaysToKeep(-1)
   }
   triggers {
-    cron('H H(3-4) * * 1-5')
+    cron('0 */1 * * 1-5')
```

Comment:
It's a cached approach? Otherwise, I'd say to run less often? Or run between a time range? I guess AMAC and EMEA are the interested ones, while APAJ might be a bit less used? Just an idea